tests/integration-tests/utils.py

```diff
@@ -125,11 +125,11 @@ def get_compute_nodes_instance_ids(stack_name, region):
         raise
 
 
-def get_instance_ids_to_compute_hostnames_dict(instance_ids):
+def get_instance_ids_to_compute_hostnames_dict(instance_ids, region):
```

Comment:
Since this is in a utils module, what about making this new arg an optional with a default of None?